codemagic.yaml

```diff
@@ -127,7 +167,7 @@ workflows:
     environment:
       java: 17
       groups:
-        - keystore_credentials
+        - keystore_credentials_project
```

Comment:
My suggestion here: Remove all the variables from Codemagic side and hardcode them into the scripts. We don't need to change them and we don't need to hide them as they are only used for testing. Makes managing all these scripts way easier if all the values are in the scripts. No need to go through Codemagic to find the values